### PR TITLE
feat(meetings): fix func calls and bind those functions

### LINKED ISSUE #1499: bug(mafia): Inconsistency with Hooker and Janitor receiving Carols
Problem:
- Hooker and Janitor (and likely other Mafia-aligned roles) receive carols on an inconsistent basis even when conducting visits during the night.

Acceptance criteria:
- If Hooker or Janitor do not perform a visit (voting no-one in the Mafia meeting and not visiting in their personal meetings) then they should receive carols. If they do visit, then they should not receive carols 100% of the time.
- Avoid a complete overhaul of visiting and meeting mechanics that result in damaging other edge cases with visiting roles.

### PR DESCRIPTION
**Pull request by dmitriigaidarji**
makes all members of mafia night meeting appear in the actors array
closes #1499

### CODE REVIEW

File: Games/types/Mafia/roles/cards/WinWithMafia.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ module.exports = class WinWithMafia extends Card {
 
         // win by majority
         const hasMajority =
-          counts["Mafia"] + numBenandanteAlive >= aliveCount / 2 &&
+          counts["Mafia"] >= aliveCount / 2 &&
```
Comment (dmitriigaidarji) [inline Games/types/Mafia/roles/cards/WinWithMafia.js:23]: @Golbolco  this is an unrelated bug in current master branch. numBenandanteAlive is not defined. modify at your will

File: Games/types/Mafia/roles/cards/MeetingMafia.js [hunk 2]
```diff
@@ -9,7 +9,7 @@ module.exports = class MeetingMafia extends Card {
       Mafia: {
         actionName: "Mafia Kill",
         states: ["Night"],
-        flags: ["group", "speech", "voting"],
+        flags: ["group", "speech", "voting", "multiActor"],
```
Comment (dmitriigaidarji) [inline Games/types/Mafia/roles/cards/MeetingMafia.js:12]: multiActor is required to pass through this if check https://github.com/UltiMafia/Ultimafia/blob/81cb77acfe3404c4b127acc5b9b5a4f57a9e6c8e/Games/core/Meeting.js#L743

File: Games/core/Meeting.js [hunk 3]
```diff
@@ -740,9 +740,9 @@ module.exports = class Meeting {
     // Do the action
     var actor, actors;
 
-    if (!this.multiActor) actor = this.leader;
+    if (!this.multiActor) actor = this.leader();
```
Comment (dmitriigaidarji) [inline Games/core/Meeting.js:743]: pretty sure this is supposed to be a call statement and not a reference pass

File: Games/types/Mafia/roles/cards/MeetingMafia.js (hunk 2 above)
Comment (Golbolco) [reply Games/types/Mafia/roles/cards/MeetingMafia.js:12]: Hold on, just to be clear: I believe that when a multiActor flag is applied, what that would mean is that for example if 3 mafia members vote to kill a player with a Bomb, all 3 die. Is this what you are intending? What are tests showing for you?

Comment (dmitriigaidarji) [reply Games/types/Mafia/roles/cards/MeetingMafia.js:12]: @Golbolco cant run tests in my env for now. But checked manually your scenario and was ok
Screenshot 2024-07-11 at 2 10 59 PM
Without multActor, the current logic is that only the first Mafia to vote for the kill visits. 
With it, the code block goes through to put all actors in
